server.go

```diff
@@ -277,6 +282,19 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 		subscribers: make(map[uint64]*preimageSubscriber),
 	}
 
+	// Set up the External Preimage client if one is set to
+	// allow us to query for preimages from the external service
+	if cfg.Extpreimage.RPCHost != "" {
```

Comment:
Just to clarify, this means we are not requiring this host to be configured for lnd to run?

Reply:
Correct. If the user does not want to use external preimages, I don't think they should be forced to, particularly since it involves setting up another service.

Perhaps there should be some kind of error / validation that happens when creating an invoice with an external preimage if no service is configured, but I think that's best left for the future.